### PR TITLE
Clarify comment about nonce space for Note Encryption

### PR DESCRIPTION
**Pull request by tomrittervg**

### CODE REVIEW

**Comment by ebfull** [comment]
Nice! Yes, this comment is stale. It describes an older version of its implementation I believe.

ACK.

File: src/zcash/NoteEncryption.hpp [hunk 1]
```diff
@@ -37,8 +37,8 @@ class NoteEncryption {
     }
 
     // Encrypts `message` with `pk_enc` and returns the ciphertext.
-    // This can only be called twice for a given instantiation before
-    // the nonce-space runs out.
+    // This is currently only called twice for a given instantiation;
```
Comment (daira) [inline src/zcash/NoteEncryption.hpp]: Let's say "ZC_NUM_JS_OUTPUTS times" here.

**Comment by daira** [comment]
@tomrittervg if you change "twice" to "ZC_NUM_JS_OUTPUTS times", I'll merge this.

**Comment by ebfull** [comment]
@zkbot r+

**Pull request merged**